.github/workflows/countline.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python3
+import os
+import sys
+
+#get current working dir
+path = os.getcwd()
+
+#parses through files and saves to a dict
+fileNamesWithSize={}
+
+#libPath and testPath dir location
+libPath = os.path.expanduser(os.path.join(path, 'lib'))
+testPath = os.path.expanduser(os.path.join(path, 'test'))
+
+#counting lines in lib dir
+for root, dirs, files in os.walk(libPath, topdown=False):
+    for name in files:
+        fileLocation = os.path.join(root, name)
+        with open(fileLocation) as f:
+            fileNamesWithSize[fileLocation]=sum(1 for line in f if line.strip() and not line.startswith('#'))
+
+#counting lines in test dir
+for root, dirs, files in os.walk(testPath, topdown=False):
+    for name in files:
+        fileLocation = os.path.join(root, name)
+        with open(fileLocation) as f:
+            fileNamesWithSize[fileLocation]=sum(1 for line in f if line.strip() and not line.startswith('#'))   
+
+#if the line rule is voilated then value is changed to 1 
+isLineRuleVoilated = 0
+fileCount = 0
+
+for key, value in fileNamesWithSize.items():
```

Comment:
[Variable name is not PEP8 compliant](https://stackoverflow.com/questions/8908760/should-i-use-camel-case-or-underscores-in-python)